contentcuration/search/viewsets/contentnode.py

```diff
@@ -64,23 +58,7 @@ def filter_channel_list(self, queryset, name, value):
         return queryset.filter(channel_id__in=list(channel_ids))
 
     def filter_keywords(self, queryset, name, value):
-        filter_query = Q(title__icontains=value) | Q(description__icontains=value)
-        tags_node_ids = ContentNode.tags.through.objects.filter(
-            contenttag__tag_name__icontains=value
-        ).values_list("contentnode_id", flat=True)[:250]
-        # Check if we have a Kolibri node id or ids and add them to the search if so.
-        # Add to, rather than replace, the filters so that we never misinterpret a search term as a UUID.
-        # node_ids = uuid_re.findall(value) + list(tags_node_ids)
-        node_ids = uuid_re.findall(value)
-        for node_id in node_ids:
```

Comment:
Noting that this is quite useful for looking up by ids - I can understand why we'd want to remove it from here, but maybe adding an additional filter to replace this functionality would be useful?

Reply:
I agree with you. I will add uuid checking on search.